### PR TITLE
Release v0.9.5

### PR DESCRIPTION
**Pull request by prkumar**
Added
-----
- Add Python 3.8, 3.9, and 3.10 as officially supported. (#237)

Fixed
-----
- Fix ``FieldMap`` and ``PartMap`` from raising ``NoneType`` error. (#221)
- Fix Python 2.7 support. (#217)

Deprecated
----------
- Python 2.7 support will be removed in v0.10.0.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -82,7 +82,9 @@ Features
   - Built-in support for `Basic Authentication`_
   - Use existing auth libraries for supported clients (e.g., |requests-oauthlib|_)
 
-Uplink officially supports Python 2.7 & 3.3-3.7.
+Uplink officially supports Python 2.7 and 3.5+.
+
+**Note:** Python 2.7 suport will be removed in v0.10.0.
```
Comment (sthagen) [inline README.rst:87]: I started to become afraid the python2 support would continue for ever … 😺

**Review by sthagen — APPROVED** [review]
LGTM.

**Comment by prkumar** [comment]
Looks like Travis CI disabled the builds for Uplink because I've burned through Travis CI's free plan: 

> Builds have been temporarily disabled for public repositories due to a negative credit balance. 

Seems like they may have a program to request for OSS credits: https://docs.travis-ci.com/user/billing-faq/#what-if-i-am-building-open-source. I'll reach out to them later. For now, i'll 

Looking forward, might be a good opportunity to consider another CI platform, like GitHub Actions.

**Comment by prkumar** [comment]
I've contacted Travis CI support. Awaiting their reply.

I've also created #243 to consider migrating to GitHub Actions.

**Comment by prkumar** [comment]
> I've contacted Travis CI support. Awaiting their reply.
> 
> I've also created #243 to consider migrating to GitHub Actions.

**Pull request merged**